tests/cases/test_build.rs

```diff
@@ -89,3 +90,54 @@ fn test_build_with_printonly(env_name: &str) {
     );
     assert!(r.is_ok(), "built heylib (printonly = true)");
 }
+
+#[parameterized(env_name = {"default", "alpine"})]
+fn test_build_with_release_create_tar_compatible_output(env_name: &str) {
+    let state = setup();
+    if !cfg!(feature = "docker") && env_name == "alpine" {
+        return;
+    }
+
+    // Test basic build functionality with heylib component
+    let component_dir = clone_component_dir("heylib", &state);
+
+    let r = fetch::fetch_input(&component_dir, &env_name, &state.backend);
+    assert!(r.is_ok(), "installed heylib dependencies: {:?}", r);
+
+    let r = build::build_for_release(&component_dir, &env_name, &state.tempdir.path(), "1");
+    assert!(r.is_ok(), "built heylib release: {:?}", r);
+
+    // Check if tar can extract the artifact archive
+    let artifact = state.tempdir.path().join("heylib/ARTIFACT/heylib.tar.gz");
+    let extracted = state.tempdir.path().join("EXTRACTED");
+    std::fs::create_dir_all(&extracted).unwrap();
+
+    let args: Vec<&str> = vec!["xvf", &artifact.to_str().unwrap(), "-C", "EXTRACTED"];
+
+    // Spawn `tar` in a subprocess to test if it can read the artifact
+    let r = Command::new("tar")
```

Comment:
funnily enough this pr removes the dependency on tar (we could remove the configure.rs check), but introduces a dev-dependency on tar in the test :D